test/functional/aws_standalone_vpc.go

```diff
@@ -0,0 +1,573 @@
+		rdsSubnetGroupsError: []string{},
+	}
+
+	// get rds subnet groups by subnet group name
+	describeGroups, err := rdsSess.DescribeDBSubnetGroups(&rds.DescribeDBSubnetGroupsInput{
+		DBSubnetGroupName: aws.String(name),
+	})
+	if err != nil {
+		newErr.rdsSubnetGroupsError = append(newErr.rdsSubnetGroupsError, fmt.Sprintf("error describing rds subnet groups: %v", err))
+		return newErr
+	}
+
+	// expect 1 subnet group
+	subnetGroups := describeGroups.DBSubnetGroups
+	if len(subnetGroups) != 1 {
+		newErr.rdsSubnetGroupsError = append(newErr.rdsSubnetGroupsError, fmt.Sprintf("unexpected number of rds subnet groups: %d", len(subnetGroups)))
+		return newErr
+	}
+
+	// ensure all subnets exist in subnet group
+	subnetsExist := true
+	for _, subnet := range subnetGroups[0].Subnets {
+		if !contains(subnets, subnet.SubnetIdentifier) {
+			subnetsExist = false
+			break
+		}
+	}
+	if !subnetsExist {
+		newErr.rdsSubnetGroupsError = append(newErr.rdsSubnetGroupsError, "rds subnet group does not contain expected subnets")
+	}
+
+	return newErr
+}
+
+// verify that the subnet groups for elasticache are created
+func verifyCacheSubnetGroups(cacheSvc *elasticache.ElastiCache, name string, subnets []*string) error {
+	newErr := &networkConfigTestError{
+		cacheSubnetGroupsError: []string{},
+	}
+
+	// get elasticache subnet groups by subnet group name
+	describeCacheGroups, err := cacheSvc.DescribeCacheSubnetGroups(&elasticache.DescribeCacheSubnetGroupsInput{
+		CacheSubnetGroupName: aws.String(name),
+	})
+	if err != nil {
+		newErr.cacheSubnetGroupsError = append(newErr.cacheSubnetGroupsError, fmt.Sprintf("error describing elasticache subnet groups: %v", err))
+		return newErr
+	}
+
+	// expect 1 subnet group
+	cacheSubnetGroups := describeCacheGroups.CacheSubnetGroups
+	if len(cacheSubnetGroups) != 1 {
+		newErr.cacheSubnetGroupsError = append(newErr.cacheSubnetGroupsError, fmt.Sprintf("unexpected number of elasticache subnet groups: %d", len(cacheSubnetGroups)))
+	}
+
+	// ensure all subnets exist in subnet group
+	subnetsExist := true
+	for _, subnet := range cacheSubnetGroups[0].Subnets {
+		if !contains(subnets, subnet.SubnetIdentifier) {
+			subnetsExist = false
+			break
+		}
+	}
+	if !subnetsExist {
+		newErr.cacheSubnetGroupsError = append(newErr.cacheSubnetGroupsError, "elasticache subnet group does not contain expected subnets")
+	}
+
+	return newErr
+}
+
+// verify that the peering connection we create has the correct requester info
+func verifyPeeringConnection(session *ec2.EC2, clusterTag, expectedCidr, vpcID string) (*ec2.VpcPeeringConnection, error) {
+	newErr := &networkConfigTestError{
+		peeringConnError: []string{},
+	}
+
+	// filter the peering connections by name and integreatly tag
+	peeringConn, err := session.DescribeVpcPeeringConnections(&ec2.DescribeVpcPeeringConnectionsInput{
+		Filters: []*ec2.Filter{
+			{
+				Name:   aws.String("tag:Name"),
+				Values: []*string{aws.String(peeringTagName)},
+			},
```

Comment:
As this tag is basically just a display name, I think we can ignore it in the test. Our provision logic doesn't depend on it, and it could easily change in the future

Reply:
Ok, does that apply to all of the other resources as well then?

Reply:
Yes, I think so. I'd be a bit concerned that our tests become so exact that we can't make metadata updates without having to update an e2e test in RHMI. It's nothing major, but I don't think these are worth testing.

Reply:
Sure, makes sense